### PR TITLE
Feature/pageheader sidebutton

### PR DESCRIPTION
**Pull request by kevinch**
screen shot 2018-08-17 at 11 03 10

### CODE REVIEW

**Comment by kevinch** [comment]
closes https://github.com/vtex/styleguide/issues/211

File: react/components/PageHeader/index.js [hunk 1]
```diff
@@ -51,6 +65,8 @@ PageHeader.propTypes = {
   title: PropTypes.string.isRequired,
   linkLabel: PropTypes.string,
   onLinkClick: PropTypes.func,
+  buttonLabel: PropTypes.string,
+  onButtonClick: PropTypes.func,
```
[the remark this answers is not in the document]
Comment (akaFTS) [reply react/components/PageHeader/index.js]: Eu gosto de `onButtonClick` :(

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [5.5.2] - 2018-08-13
+
+### Added
```
Comment (klzns) [inline CHANGELOG.md]: Colocar dentro de `[Unreleased]`

File: react/components/PageHeader/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kevinch) [reply react/components/PageHeader/index.js]: potato patato...

**Pull request merged**